ansible-container/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rhel7
+
+MAINTAINER Aaron Weitekamp <aweiteka@redhat.com>
+
+RUN yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+# Not sure if all of these packages are necessary
+# only git and ansible are known requirements
+RUN yum install -y --enablerepo rhel-7-server-extras-rpms net-tools bind-utils tmux git ansible
+
+RUN git clone https://github.com/openshift/openshift-ansible.git /opt/openshift-ansible
```

Comment:
@aweiteka I am trying to understand this. It seems to me that the above line adds everything inside / on the host into /opt/openshift-ansible/ in the building container. Is that desired?